core/dplug/core/ringbuf.d

```diff
@@ -128,8 +128,12 @@ struct RingBufferNoGC(T)
         T opIndex(size_t index) nothrow @nogc
         {
             // crash if index out-of-bounds (not recoverable)
-            if (index > _count)
-                assert(0);
+            version(D_NoBoundsChecks) {}
```

Comment:
I'd guess it's nice to respect that compiler switch.

Reply:
(Please don't get fooled by GitHub not showing the newly added boundscheck in this box.)